README.md

```diff
@@ -51,23 +43,77 @@ To build the project execute the following command:
       {"status":"UP","diskSpace":{"status":"UP","total":249644974080,"free":137188298752,"threshold":10485760}}
     ```
 
-- To run all functional tests or single test you can run as Junit, make sure the env is set
-    ```
-        OPEN_ID_IDAM_URL=http://'minikubeIP:port'
-    ```
-  Note: Make sure the BPMN and DMN are deployed onto Camunda locally. Workflow and Task Configuration services should be running
-
-- To run all tests including junit, integration and functional.
-  NOTE: This service is dependent on wa-workflow-api and wa-task-configuration service , so make sure it is running locally when running FTs.
-
-  You can run the command
-   ```
-       ./gradlew test integration functional
-   ```
-  or
-  ```
-  ./gradlew tests
-  ```
+### Running tests
+
+#### Pre-requisites
+
+Case Event Handler is dependent on the following services running locally (alongside minikube containers) when running
+functional tests 
+
+- WorkAllocation Services
+    - [wa-workflow-api](https://github.com/hmcts/wa-workflow-api)
+    - [wa-task-management-api](https://github.com/hmcts/wa-task-management-api)
+
+- Immigration/Asylum Services
+    - [ia-case-documents-api](https://github.com/hmcts/ia-case-documents-api)
+    - [ia_case_api](https://github.com/hmcts/ia_case_api)
+    - [ia-case-notifications-api](https://github.com/hmcts/ia-case-notifications-api)
+    
+Ensure the IU ccd case definitions are uploaded by installing yarn and running the script
```

Comment:
It is IA ccd case definition, isn't it

Reply:
oops, yes, good spot

Reply:
Fixed in [34e8688](https://github.com/hmcts/wa-case-event-handler/pull/241/commits/34e8688e3ebe8e7314a1f0b8ab2114fa9f2fd12e)